community.md

```diff
@@ -63,7 +63,9 @@ Subscribe to our [mailing list](https://mail.mozilla.org/listinfo/dev-addons) de
 
 ### Questions about add-on development
 
-You can get help and support by posting to [Mozilla's community forum](https://discourse.mozilla.org/c/add-ons), [Stack Overflow](http://stackoverflow.com/questions/tagged/firefox-addon), or the [developer mailing list](https://mail.mozilla.org/listinfo/dev-addons).
+You can get help and support by posting to [Mozilla's community forum](https://discourse.mozilla.org/c/add-ons), [Matrix Chat Rooms](https://wiki.mozilla.org/Matrix): [#add-ons](https://bit.ly/2PKCHuH), [#addons-reviewers](https://bit.ly/32Odp4i), [Stack Overflow](http://stackoverflow.com/questions/tagged/firefox-addon), or the [developer mailing list](https://mail.mozilla.org/listinfo/dev-addons).
+
+If you already use Matrix, then connect to *#addon-reviewers:mozilla.org*. If you have a Firefox account you can use [chat.mozilla.org/#/room/#addon-reviewers:mozilla.org](https://mzl.la/2x5jNs1) .
 
```

Comment:
Don't forget to update the "last updated" and "updated by" metadata on the page! 😉

![Screen Shot 2020-03-03 at 12 40 45 PM](https://user-images.githubusercontent.com/23342476/75817621-4bde9f00-5d4c-11ea-8303-a5edf7af7f19.png)